.githooks/pre-commit

```diff
@@ -18,6 +18,9 @@
 files=`git diff --cached --name-only`
 year=`date +%Y`
 
+git rev-parse -q --verify MERGE_HEAD && \
```

Comment:
It just checks if the `MERGE_HEAD` exists so git is in the middle of merge, correct?

Reply:
that's how I understand it

Reply:
ok